airflow/providers/google/provider.yaml

```diff
@@ -744,9 +744,22 @@ hook-class-names:
   - airflow.providers.google.cloud.hooks.cloud_sql.CloudSQLHook
   - airflow.providers.google.cloud.hooks.cloud_sql.CloudSQLDatabaseHook
   - airflow.providers.google.cloud.hooks.compute_ssh.ComputeEngineSSHHook
-  - airflow.providers.google.cloud.hooks.bigquery.BigQueryHook
```

Comment:
This hook here made no sense. It's connection_type was the same as GoogleBaseHook.

Reply:
I actually brought it back with different connection type.